src/resolvers/query/workspace-user/index.js

```diff
@@ -0,0 +1,25 @@
+import { userFragment } from "./fragment";
+import { addFragmentToInfo } from "graphql-binding";
+import { find } from "lodash";
+/*
+ * Get a single user on a workspace.
+ * @param {Object} parent The result of the parent resolver.
+ * @param {Object} args The graphql arguments.
+ * @param {Object} ctx The graphql context.
+ * @return users on this specific workspace
+ */
+export default async function workspaceUser(parent, args, ctx, info) {
+  const workspaceUsers = await ctx.db.query.users(
+    {
+      where: {
+        roleBindings_some: {
+          workspace: { id: args.workspaceUuid }
+        }
```

Comment:
```suggestion
        },
        username: args.username
```

Then we don't need to filter the user in JS but can let the server handle it. (I think that works)